app/connectors/httpParsers/DESParser.scala

```diff
@@ -30,9 +31,9 @@ trait DESParser {
     Some(s"[$parserName][read] Received ${response.status} from DES. Body:${response.body}" + getCorrelationId(response))
   }
 
-  def badSuccessJsonFromDES[Response]: Either[DesErrorModel, Response] = {
-    pagerDutyLog(BAD_SUCCESS_JSON_FROM_DES, Some(s"[$parserName][read] Invalid Json from DES."))
-    Left(DesErrorModel(INTERNAL_SERVER_ERROR, DesErrorBodyModel.parsingError))
+  def badSuccessJsonFromDES[Response](validationErrors: Seq[(JsPath, Seq[JsonValidationError])] ): Either[DesErrorModel, Response] = {
+    pagerDutyLog(BAD_SUCCESS_JSON_FROM_DES, Some(s"[$parserName][read] Invalid Json from DES. " + validationErrors))
+    Left(DesErrorModel(INTERNAL_SERVER_ERROR, DesErrorBodyModel("PARSING_ERROR", "Error parsing response from DES - " + validationErrors)))
```

Comment:
optional, string constant or relevant enum need to be defined for PARSING_ERROR, in same fashion as for BAD_SUCCESS_JSON_FROM_DES and INTERNAL_SERVER_ERROR

Reply:
yep agreed - future PR perhaps